script.js

```diff
@@ -0,0 +1,129 @@
+const tttBox = document.querySelectorAll(".ttt_square")
+const playerOneScore = document.querySelector(".p1score")
+const playerTwoScore = document.querySelector(".p2score")
+const winnerText = document.querySelector(".winner_text")
+const playerOneArrow = document.querySelector(".arrowp1")
+const playerTwoArrow = document.querySelector(".arrowp2")
+const playerOnePiece = "X"
+const playerTwoPiece = "O"
+let playerOneTurn = true;
+let playerTwoTurn = false;
```

Comment:
To clean up a bit, you can just have playerOneTurn = true and when it's false it can be assumed that it is player 2's turn so you don't need to define playerTwo and reassign